spec/controllers/coronavirus_form/offer_care_controller_spec.rb

```diff
@@ -55,11 +55,17 @@
       expect(response).to redirect_to(offer_other_support_path)
     end
 
-    it "clears previously entered care cost for a 'No' response" do
+    it "clears previously entered answers given a 'No' response" do
+      session[:offer_care_type] = I18n.t("coronavirus_form.questions.offer_care_qualifications.offer_care_type.options").map { |_, item| item[:label] }.sample
```

Comment:
I'd suggest a better pattern to `.map { |_, item| .... }` is `.values.map { |item| }`

Reply:
Thanks @reggieb - good spot 👍  

I agree, just mapping the values makes more sense than defining a useless and subsequently ignored variable.  However, as the `.map { |_, item| .... }` pattern is used throughout the form, so I'll go with the `prior art` approach.